plugins/scaffolder/src/next/OngoingTask/OngoingTask.tsx

```diff
@@ -100,6 +118,14 @@ export const OngoingTask = (props: {
     templateRouteRef,
   ]);
 
+  const [{ status: cancelStatus }, { execute: triggerCancel }] = useAsync(
+    async () => {
+      if (taskId) {
+        await scaffolderApi.cancelTask(taskId);
+      }
+    },
```

Comment:
should this have a `, [taskId]` on it? otherwise you can't be entirely sure it bound to the most current value of the id

Reply:
Yeah, that makes sense to me but I can't seem to do that with the `@react-hookz/web` version of `useAsync`. Any suggestions?

Reply:
Aha, then maybe it just plain works differently than the other hooks typically have done. Hm.